closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -867,7 +1103,10 @@ JNIEXPORT jlong JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_createRSAPrivat
  */
 JNIEXPORT void JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_destroyRSAKey
   (JNIEnv *env, jclass obj, jlong rsaKey) {
-    RSA_free((RSA*)rsaKey);
+    RSA* rsaKey2 = (RSA*) rsaKey;
```

Comment:
This should also have a cast through `(intptr_t)`.

Reply:
Fixed, hopefully that's all of them..

Reply:
I didn't see any others. Thanks.